docs/purpose.md

```diff
@@ -5,7 +5,8 @@ Today, the primary purpose of Online Boutique is to demonstrate:
 * [Google Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine)
 * [Anthos](https://cloud.google.com/anthos)
 * [Google Cloud Operations](https://cloud.google.com/products/operations)
-* tools and technologies commonly used alongside the above products
+* tools, technologies, and patterns commonly used alongside the above products
+* tools, technologies, and patterns commonly used by the retail industry
```

Comment:
This could open up opportunity doors to quite a lot of non-Google-Cloud tools due to the loose association to "used by the retail industry". Is that intentional?

Reply:
Potentially still has "loopholes" but at the end of the day the owners of this app have the final say on new feature requests, regardless of this doc. So I think it's fine like this!